snapshot_test.go

```diff
@@ -37,17 +38,19 @@ func TestSnapshotCreate(t *testing.T) {
 	testPassword := "this_is_a_password"
 
 	tests := []struct {
-		compression uint16
-		parityParts uint
+		compression     uint16
+		parityParts     uint
+		excludesStore   []string
+		excludesRestore []string
 	}{
-		{CompressionNone, 0},
-		{CompressionFlate, 0},
-		{CompressionGZip, 0},
-		{CompressionLZMA, 0},
-		{CompressionZlib, 0},
-		{CompressionZstd, 0},
-		{CompressionNone, 1},
-		{CompressionGZip, 1},
+		{CompressionNone, 0, []string{"snapshot.go"}, []string{}},
+		{CompressionFlate, 0, []string{"snapshot_test.go"}, []string{}},
+		{CompressionGZip, 0, []string{"snapshot.go", "snapshot_test.go"}, []string{}},
+		{CompressionLZMA, 0, []string{}, []string{"snapshot.go"}},
+		{CompressionZlib, 0, []string{}, []string{"snapshot_test.go"}},
+		{CompressionZstd, 0, []string{}, []string{"snapshot.go", "snapshot_test.go"}},
+		{CompressionNone, 1, []string{"snapshot.go"}, []string{"snapshot_test.go"}},
+		{CompressionGZip, 1, []string{}, []string{}},
```

Comment:
I'd like to keep the original compression tests without any excludes, and then add the exclude tests separately, once with `CompressionNone` and once with `CompressionGZip`.